napari/_qt/layers/qt_labels_layer.py

```diff
@@ -230,16 +277,38 @@ def _on_n_dim_change(self, event=None):
             self.ndimCheckBox.setChecked(self.layer.n_dimensional)
 
     def _on_contig_change(self, event=None):
+        """Receive layer model contiguous change event and update contiguous checkbox.
+
+        Parameters
+        ----------
+        event : qtpy.QtCore.QEvent, optional.
+            Event from the Qt context.
+        """
         with self.layer.events.contiguous.blocker():
             self.contigCheckBox.setChecked(self.layer.contiguous)
 
     def _on_editable_change(self, event=None):
+        """Receive layer model editable change event and enable/disable editing buttons.
+
+        Parameters
+        ----------
+        event : qtpy.QtCore.QEvent, optional.
+            Event from the Qt context.
+        """
         self.pick_button.setEnabled(self.layer.editable)
         self.paint_button.setEnabled(self.layer.editable)
         self.fill_button.setEnabled(self.layer.editable)
 
 
 class QtColorBox(QWidget):
+    """QtColorBox class
+
```

Comment:
The QtColorBox class needs a better one line summary, what should we put here?

Reply:
```suggestion
    """A widget that shows a square with the current label color.
```